### ISSUE TITLE
Missing Form label

### ISSUE THREAD

**Issue by spencercheng** [BYU-ODH/wp-humanities-byu#462]
The following needs a `label` tag because it has an `input` tag: 

- [x] The search bar under the project's page.
- [ ] The search bar under the people's page.
